packages/terra-functional-testing/lib/terra-dev-site/doc/asset-service/AssetService.doc.mdx

```diff
@@ -0,0 +1,199 @@
+
+To use the service it must be added to the services array in the `wdio.conf.js` file.
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+            // Asset service options here.
+            // ...
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+### config
+
+A path to a webpack configuration file.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: The asset service will check the current working directory for a `webpack.config.js` file.
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          config: './path/webpack.config.js'
+        }]
+    ],
+    // ...
+};
+```
+
+### index
+
+The asset index.
+
+Type: `string`
+
+Required: `false`
+
+Default: `index.html`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          index: 'index.html',
+        }]
+    ],
+    // ...
+};
+```
+
+### locale
+
+An optional locale that will be passed into the webpack configuration as an environment variable.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          locale: 'en-US',
+        }]
+    ],
+    // ...
+};
+```
+
+### port
+
+The port the service will be hosted on.
+
+Type: `number`
+
+Required: `false`
+
+Default: `8080`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          port: 8080,
+        }]
+    ],
+    // ...
+};
+```
+
+### site
+
+A relative path to a directory of static assets. If provided, an express server will be launched to serve the directory assets.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetService, {
+          site: './build',
+        }]
+    ],
+    // ...
+};
+```
+
+### theme
+
+An optional theme that will be passed into the webpack configuration as an environment variable.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
```

Comment:
Should this be terra-default-theme?

Reply:
I updated this to default to the terra-default-theme. We'll need to evaluate down the road any quirks this might have

https://github.com/cerner/terra-toolkit/pull/417/commits/9d536c1de47e906b3e6248b5bdb0ce0b5b0a8087